fsociety/core/config.py

```diff
@@ -26,6 +27,7 @@
     "os": CURRENT_PLATFORM,
     "host_file": "hosts.txt",
     "usernames_file": "usernames.txt",
+    "development": "false",
```

Comment:
I think it may be best to use debug levels?

Reply:
sure, i'll try doing log level instead and set that on init for the app.